From c06937ddfc837e59a634557891db2c78357d8add Mon Sep 17 00:00:00 2001
From: You-Sheng Yang <vicamo@gmail.com>
Date: Fri, 26 Aug 2016 18:38:47 +0800
Subject: [PATCH 3/3] build: fix a compile warning

urf_daemon_block() tries to return a value from as a void function.

Index: urfkill-0.6.0~20150318.103828.5539c0d/src/urf-daemon.c
===================================================================
--- urfkill-0.6.0~20150318.103828.5539c0d.orig/src/urf-daemon.c
+++ urfkill-0.6.0~20150318.103828.5539c0d/src/urf-daemon.c
@@ -294,7 +294,7 @@ urf_daemon_block (UrfDaemon
 	char *error_str;
 	gboolean done = FALSE;
 
-	g_return_val_if_fail (type >= 0, FALSE);
+	g_return_if_fail (type >= 0);
 
 	if (!urf_arbitrator_has_devices (priv->arbitrator))
 		goto out;
